hamilton/plugins/pandas_extensions.py

```diff
@@ -11,6 +11,8 @@
 except ImportError:
     raise NotImplementedError("Pandas is not installed.")
 
+from sqlite3 import Connection
```

Comment:
we could probably do a try catch here on import exception and provide something the works in case sqllite isn't present.. I think something like this should work:
```suggestion
try:
   from sqlite3 import Connection
except ImportError:
   Connection = object
```

Reply:
TIL: I think py3.7-3.11 already includes SQLite by default, so I'll probably remove the try/except block around the SQLite3 import.